contrib/epee/include/net/http_client.h

```diff
@@ -321,8 +322,9 @@ namespace net_utils
 				m_ssl_support = ssl_support;
 				m_ssl_private_key_and_certificate_path = private_key_and_certificate_path;
 				m_ssl_allowed_certificates = allowed_ssl_certificates;
+                m_ssl_allowed_fingerprints = allowed_ssl_fingerprints;
```

Comment:
It would be nice to keep the same space/tabs as the surrounding code so this does not look out of kilter :)